dist/stat_data.py

```diff
@@ -336,6 +336,22 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     SessionStat('session_cursor_open', 'open cursor count', 'no_clear,no_scale'),
     SessionStat('session_open', 'open session count', 'no_clear,no_scale'),
+    SessionStat('session_table_compact_fail', 'table compact failed attempts', 'no_clear,no_scale'),
```

Comment:
It would be more consistent to say `calls` instead of `attemtps` for all of these stats.  See AsyncStats and CursorStats for precedent.